src/App.tsx

```diff
@@ -45,6 +46,7 @@ function App() {
           <Route path="whiskey" element={<WhiskeyCatalog />} />
         </Routes>
       </main>
+      <ErrorPage/>
```

Comment:
please add a Route for this page
the sample is here: 
![image](https://github.com/mikailcfrli/wine-and-dine/assets/18329725/c380fe84-ea19-4903-ba27-79749e2c04b7)